Importing Official TL s from Matsuribayashi (#372)
07th-mod/higurashi-console-arcs: Update/omot_021.txt

//r…お、それはいいたとえだな！　真っ赤に燃える男、前原圭一！！　青く燃える女、竜宮レナ！　我が部にその人ありと天下に知らしめる絶好のチャンスだぜ！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "…お、それはいいたとえだな！　真っ赤に燃える男、前原圭一！！　青く燃える女、竜宮レナ！　我が部にその人ありと天下に知らしめる絶好のチャンスだぜ！！",
			NULL, "...Hey, that's a good one! The Burning Red Ranger, Keiichi Maebara!! The Burning Blue Ranger, Rena Ryuugu! This is our chance to show the world what we've got!!", Line_Normal);
	ClearMessage();

//rそして、それに加えてわが部活メンバー最強のツートップが要所を締める！　個にして最強、揃えば無敵！！　それが俺たちの部だッ！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "そして、それに加えてわが部活メンバー最強のツートップが要所を締める！　個にして最強、揃えば無敵！！　それが俺たちの部だッ！！",

> 			NULL, "And in addition to that, our club members also have the greatest tag team duo! We're already powerful individually, and when gather, we're invincible!!", Line_Normal);

Reviewer: ```suggestion
			NULL, "And in addition to that, our club members also have the greatest tag team duo! We're already powerful individually, and together, we're invincible!!", Line_Normal);
```

			NULL, "\"...Heh heh heh heh.", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300083", 256, TRUE);
	OutputLine(NULL, "いやはや、…不謹慎だと思って黙ってたけど、おじさんさ、本当はこう思ってたのさ。",
			NULL, " You know... I kept quiet, thinking it might be imprudent, but actually I've been thinking...", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300084", 256, TRUE);
	OutputLine(NULL, "…こいつぁ最高に面白くなってきた、ってねッ！！」",

> 			NULL, " ...this is getting super interesting!!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " this is getting super interesting!!\"", Line_Normal);
```

	ClearMessage();

//魅音rvS14/03/220300085.「今は感謝してるよ梨花ちゃん。kvS14/03/220300086.これだけの獲物をよくぞ知らせてくれたッ！！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300085", 256, TRUE);
	OutputLine(NULL, "「今は感謝してるよ梨花ちゃん。",

> 			NULL, "\""I appreciate this, Rika-chan.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"I appreciate this, Rika-chan.", Line_WaitForInput);
```

	OutputLine(NULL, "これだけの獲物をよくぞ知らせてくれたッ！！」",
			NULL, " Thank you for giving us this opportunity!!\"", Line_Normal);
	ClearMessage();

//魅音rvS14/03/220300087.「くっくっく、この雛見沢で我が部に挨拶なしで上等を決めてくれようとはね！kvS14/03/220300088.　きっちりケジメを取らせてもらうよ！！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300087", 256, TRUE);
	OutputLine(NULL, "「くっくっく、この雛見沢で我が部に挨拶なしで上等を決めてくれようとはね！",
			NULL, "\"Heh heh heh, how dare they even think of stepping into Hinamizawa without saying hello to us!", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300088", 256, TRUE);
	OutputLine(NULL, "　きっちりケジメを取らせてもらうよ！！」",
			NULL, " I'll make sure to teach them some manners!!\"", Line_Normal);
	ClearMessage();

//rみな、一斉に盛り上がって腕を撫す。俺たちの士気は今や最高潮だ。…とてつもない陰謀が相手なのに、みんなと一緒なら全然負ける気がしないんだ。…こいつぁ、最高の気分だぜッ！！
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	if (GetGlobalFlag(GADVMode)) { OutputLine(NULL, "", NULL, "<size=-2>", Line_Continue); }
	OutputLine(NULL, "みな、一斉に盛り上がって腕を撫す。俺たちの士気は今や最高潮だ。…とてつもない陰謀が相手なのに、みんなと一緒なら全然負ける気がしないんだ。…こいつぁ、最高の気分だぜッ！！",

> 			NULL, "Everyone was eager to show their paces and getting excited all at once. our morale was already at its peak. ...We're up against such an absurd conspiracy, but if we're together, I think we can defeat it. ...This is the best feeling ever!!", Line_Normal);

Reviewer: ```suggestion
			NULL, "Everyone was eager to show their paces and getting excited all at once. Our morale was already at its peak. ...We're up against such an absurd conspiracy, but if we're together, I think we can defeat it. ...This is the best feeling ever!!", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 2, "ps3/s14/02/220200058", 256, TRUE);
	OutputLine(NULL, "「かなり危険な一手だね。",
			NULL, "\"That really is dangerous...", Line_WaitForInput);
	ModPlayVoiceLS(4, 2, "ps3/s14/02/220200059", 256, TRUE);
	OutputLine(NULL, "…でも確かに、鷹野が梨花ちゃんを追っているという大きな証拠を掴める手だよ」",

> 			NULL, " ...But that would prove Takano-san is after Rika-chan.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " But that would prove Takano-san is after Rika-chan.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#fcdb77>沙都子</color>", NULL, "<color=#fcdb77>Satoko</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 4, "ps3/s14/04/220400060", 256, TRUE);
	OutputLine(NULL, "「そうですわね。",

> 			NULL, "\"She's true.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"She's right.", Line_WaitForInput);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300121", 256, TRUE);
	OutputLine(NULL, "「うん。裏山篭城は追い詰められた最後の切り札になるだろうね。",
			NULL, "\"Hmm. The barricade on the hill is our last resort, huh?", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300122", 256, TRUE);
	OutputLine(NULL, "これは本当に最後の手段。",
			NULL, " The very final resort", Line_WaitForInput);
	ModPlayVoiceLS(4, 3, "ps3/s14/03/220300123", 256, TRUE);
	OutputLine(NULL, "ウチに篭城していても敵に襲われ、拠点として捨てざるを得ない時だけのね」",

> 			NULL, " for when the enemy attacks the us barricaded in my house, and we have no other choice but to leave.\"", Line_Normal);

Reviewer: ```suggestion
			NULL, " for when the enemy attacks us barricaded in my house, and we have no choice but to leave.\"", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 2, "ps3/s14/02/220200063", 256, TRUE);
	OutputLine(NULL, "「うん。きっとあるよ。",
			NULL, "\"Yeah, I'm sure we can.", Line_WaitForInput);
	ModPlayVoiceLS(4, 2, "ps3/s14/02/220200064", 256, TRUE);
	OutputLine(NULL, "私たちは梨花ちゃんが思ってるほど無力じゃないし、切り札だってないわけじゃない。",

> 			NULL, " I'm sure we can. We're not as powerless as you think, Rika-chan, and it's not like we don't have cards to play.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, " We're not as powerless as you think, Rika-chan, and it's not like we don't have cards to play.", Line_WaitForInput);
```
